decisions/DR005-String-encoding.md

```diff
@@ -0,0 +1,12 @@
+# DR005 String encoding
+
+- **Status:** Decided
+- **Impact:** Low
+- **Driver:** @foundrytom
+- **Approver:** @foundrytom, @feltech
+- **Outcome:** OpenAssetIO will use UTF-8 string encoding wherever applicable.
+
+## Relevant data
+
+[https://utf8everywhere.org](https://utf8everywhere.org)
```

Comment:
[question] Just to double-check, is this the complete decision record?  I agree it's probably sufficient to just link utf8everywhere, but thought it worth checking there aren't any missing commits or anything.